feature/visited/src/main/kotlin/uk/govuk/app/visited/ui/VisitedScreen.kt

```diff
@@ -57,22 +71,64 @@ private fun VisitedScreen(
     onPageView: () -> Unit,
     onBack: () -> Unit,
     onClick: (title: String, url: String) -> Unit,
+    onEditClick: () -> Unit,
     modifier: Modifier = Modifier
 ) {
     LaunchedEffect(Unit) {
         onPageView()
     }
 
     val title = stringResource(R.string.visited_items_title)
+    val editText = stringResource(R.string.visited_items_edit_button)
 
     Column(modifier) {
-        ChildPageHeader(
-            text = title,
-            onBack = onBack
-        )
-        LazyColumn(Modifier.padding(horizontal = GovUkTheme.spacing.medium)) {
+        val visitedItems = uiState?.visited
+
+        Column(modifier) {
+            Row(
+                modifier = Modifier.height(64.dp)
+                    .fillMaxWidth()
+                    .padding(end = GovUkTheme.spacing.small),
+                verticalAlignment = Alignment.CenterVertically,
+            ) {
+                TextButton(
+                    onClick = onBack,
+                ) {
+                    Icon(
+                        imageVector = Icons.AutoMirrored.Filled.ArrowBack,
+                        contentDescription = stringResource(uk.govuk.app.design.R.string.content_desc_back),
+                        tint = GovUkTheme.colourScheme.textAndIcons.link
+                    )
+                }
+
+                if (!visitedItems.isNullOrEmpty()) {
+                    Spacer(modifier = Modifier.weight(1f))
+
+                    TextButton(
+                        onClick = onEditClick,
+                        modifier = Modifier.padding(end = GovUkTheme.spacing.small)
+                    ) {
+                        BodyRegularLabel(
+                            text = editText,
+                            color = GovUkTheme.colourScheme.textAndIcons.link,
+                        )
+                    }
+                }
+            }
+            LargeTitleBoldLabel(
+                text = title,
+                modifier = modifier
+                    .fillMaxWidth()
+                    .height(47.dp)
+                    .padding(horizontal = GovUkTheme.spacing.medium)
+                    .padding(bottom = GovUkTheme.spacing.small)
+            )
+        }
+        LazyColumn(
+            Modifier.padding(horizontal = GovUkTheme.spacing.medium)
+                .padding(top = GovUkTheme.spacing.small)
+        ) {
             item {
-                val visitedItems = uiState?.visited
                 if (visitedItems != null) {
```

Comment:
I know this unchanged but we could do like I've mentioned elsewhere whilst we're here and change to?:

`visitedItems ?: return@item`

Or use `visitedItems?.let { .. }`

Reply:
Went with `isNullOrEmpty()` - seemed nicer and easier to read